src/backends/appkit.rs

```diff
@@ -196,6 +196,18 @@ impl Backend for AppKit {
         Ok(())
     }
 
+    #[cfg(target_arch = "aarch64")]
```

Comment:
I think the better way to do this is to assign `is_speaking` conditionally from within the function body of a single `is_speaking` function definition.

See https://github.com/ndarilek/tts-rs/blob/master/src/lib.rs#L182 for an example. Here we define `tts` conditionally based on which backend was passed into `new`. The `Default` impl for `TTS` does something similar.

Thanks!

Reply:
Cool, I'll give it another shot! I also tried to cfg macro to no avail..

Reply:
Hm I can't get it to build, maybe the confusion is that it isn't `is_speaking` that's the problem, it's `YES`. I tried this:

```rust
    fn is_speaking(&self) -> Result<bool, Error> {
        let is_speaking: i8 = unsafe { msg_send![self.0, isSpeaking] };

        if cfg!(target_arch = "aarch64") {
            Ok(is_speaking == YES as i8)
        } else {
            Ok(is_speaking == YES)
        }
    }
```

Which yields

```
error[E0308]: mismatched types
   --> src/backends/appkit.rs:205:31
    |
205 |             Ok(is_speaking == YES)
    |                               ^^^ expected `i8`, found `bool`
```

I can't figure out how you mean with assigning `is_speaking` conditionally, because the error seems to be that `YES` differs, not `is_speaking` (it's explicitly typed as `i8`).

Any more pointers? Sorry!

Reply:
I tried a new, really weird solution based on how the objc `YES` consts were set up. Probably unacceptable, but want to see if it passed your tests :)